eg/led-chars-demo.js

```diff
@@ -0,0 +1,36 @@
+var five = require("../lib/johnny-five");
+var ledChars = require("../lib/led/led-chars.js");
```

Comment:
This isn't necessary, all the characters have been defined as `five.Led.Matrix.CHARS`